### ISSUE TITLE
Does not find autoclosing tag (ie: <img />)

### ISSUE THREAD

**Issue by mallegrini** [maxhumber/gazpacho#17]
Soup.find('img') fails to find <img> tags 
**To Reproduce**

<pre>
s=Soup('&lt;div>&lt;img src="" />&lt;/div>')
print(s.find('img'))
>None
</pre>

**Expected behavior**
It should return img tag.

As a matter of fact 

<pre>
s=Soup(&lt;div>&lt;img src="" >&lt;/img>&lt;/div>') 
</pre>
works
**Environment:**
 - OS: Linux

**Comment by maxhumber** [comment]
Hey @mallegrini!

Thanks for flagging this bug. Unfortunately, this is caused by malformed HTML...

`<img>` tags aren't supposed to end with `/>` ([source](https://www.w3schools.com/tags/tag_img.asp))

Meaning that gazpacho is looking for:

```
s=Soup('<div><img src="hi.png"></div>')
print(s.find('img'))
```

But, like you, I have encountered *some* websites that contain this malformed img tag syntax (and it sucks).

I'll have to meditate on a fix... unfortunately I don't think I have a silver bullet right now :(

**Comment by maxhumber** [comment]
Fixed in Version 1.0!
